docs/src/pages/system/basics/basics.md

```diff
@@ -1,338 +1,215 @@
-The demos are also based on the **default** Material-UI [theme object](/customization/default-theme/).
+### 1. Building consistent UIs is hard
 
-{{"demo": "pages/system/basics/Demo.js", "defaultCodeOpen": true}}
+This is especially true, when there is at least more than one person building the application. There has to be some synchronization to what are the design tokens and how are those used. What parts of the theme structure should be used with what CSS properties etc.
 
-### Installation
+### 2. Switching context between JS and CSS
 
-```jsx
-// with npm
-npm install @material-ui/system
+Often we find ourselves jumping from the JS to the CSS files, or from the components definition to the instance usage in order to conclude where and how some styles are defined. This is particularly true as the complexity (LOCs/# of elements) of the component we are working on increases. We could save a lot of time by removing this contraint.
 
-// with yarn
-yarn add @material-ui/system
-```
+### 3. Less code to type
 
-### Create a component
+Usually when defining styles for a react component we need to either define a separate stylesheet, use some kind of factory for creating the component (`styled()`), or use some kind of hook for generating the styles (for example `makeStyles()` & `useStyles()`), which not only means we need to switch the context from where we are currently in the code, but we need to also type much more code than the actual styles we want to have on some element.
 
-In order to use the `Box` component, you first need to create it.
-To start with, add a `spacing` and `palette` function to the style argument.
+## How do we solve this?
 
-```jsx
-import styled from 'styled-components';
-import { spacing, palette } from '@material-ui/system';
+In order to solve these issues, we need to have a simple way of pulling & wiring the correct design tokens for specific CSS properties, and adding them directly on the react element where we want the styles to be applied, by a prop, that can be easily discoverable.
 
-const Box = styled.div`
-  ${spacing}${palette}
-`;
+The `sx` prop, as part of the system is the solution we see for solving these problems. In the example above, you may see how it can be used on the MUI components.
 
-export default Box;
-```
+The property behaves as a superset of CSS that offers mapping values from the theme directly, by pulling specific values depending on the CSS property used. In addition to this, it allows a simple way of defining responsive values, that corresponds to the breakpoints values defined in the theme.
 
-This Box component now supports new [spacing props](/system/spacing/#api) and [color props](/system/palette/#api).
-For example, you can provide a padding prop: `p` and a color prop: `color`.
+With it you can build easily your custom visual components, like `Card`, `Badge`, `Chip` that could accept the props & behave exactlly as your design system specifies.
```

Comment:
```suggestion
With it you can easily build your custom visual components, like `Card`, `Badge`, `Chip` that could accept the props & behave exactly as your design system specifies.
```

> could accept the props

What props is this refering to?

Reply:
I meant the `Card` component that they build can have it's own props, different than our `Card` component. Maybe we should just say 

```suggestion
With it you can easily build your custom visual components, like `Card`, `Badge`, `Chip` that would behave exactly as your design system specifies.
```